assemblyai/lemur.py

```diff
@@ -121,7 +133,7 @@ class Lemur:
 
     def __init__(
         self,
-        sources: List[types.LemurSource],
+        sources: Optional[List[types.LemurSource]] = None,
```

Comment:
q: Why is this changing? Was this always optional?

Reply:
It's now optional. You can provide sources or input text.